packages/salesforcedx-vscode-core/src/orgBrowser/metadataType.ts

```diff
@@ -8,124 +8,67 @@
 import {
   CliCommandExecutor,
   Command,
+  CommandOutput,
```

Comment:
The code in this file both runs a describe command and hosts utility methods (e.g. getTypesFolder) that are unrelated to the describe command. Can we create another file specific for the utility methods used across the org browser code ?

Reply:
Yeah I can do that- I'll do the same for the metadataCmp file because it also contains utility code and the list metadata command.